packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -78,8 +78,8 @@ const InsertPropertyRow = ({
   const ourInputRef = useRef<HTMLInputElement>(null);
   const sharedRef = useForkRef(inputRef, ourInputRef);
 
-  const { watch } = useFormContext<EntityTypeEditorForm>();
-  const properties = watch("properties");
+  const { control } = useFormContext<EntityTypeEditorForm>();
+  const properties = useWatch({ control, name: "properties" });
```

Comment:
I guess this prevents re-rendering by memoizing `properties`?

Reply:
Using `watch` creates a subscription in the component using `useForm`, rather than the component subscribing. Which makes sense when you think about how hooks work. This means the whole form will re-render when anything changes. We use `useWatch` to limit the re-render to this component.